src/app/common/money/calculate-money.ts

```diff
@@ -9,7 +9,7 @@ import { formatMoney } from './format-money';
 import { isMoney } from './is-money';
 
 export function baseCurrencyAmountInQuote(quantity: Money, { pair, price }: MarketData) {
-  if (quantity.symbol !== pair.base)
+  if (quantity.symbol.toLowerCase() !== pair.base.toLowerCase())
```

Comment:
We get some data back that doesn't always have the right casing here for symbols, so thinking best to force them all lowercase for this check?